static/app/views/issueList/groupSearchViewTabs/issueViews.tsx

```diff
@@ -16,16 +24,18 @@ import normalizeUrl from 'sentry/utils/url/normalizeUrl';
 import {useNavigate} from 'sentry/utils/useNavigate';
 import useOrganization from 'sentry/utils/useOrganization';
 import usePageFilters from 'sentry/utils/usePageFilters';
-import {generateTempViewId} from 'sentry/views/issueList/groupSearchViewTabs/draggableTabBar';
 import {useUpdateGroupSearchViews} from 'sentry/views/issueList/mutations/useUpdateGroupSearchViews';
 import type {
   GroupSearchView,
   UpdateGroupSearchViewPayload,
 } from 'sentry/views/issueList/types';
 import {IssueSortOptions} from 'sentry/views/issueList/utils';
+import {NewTabContext, type NewView} from 'sentry/views/issueList/utils/newTabContext';
 
 const TEMPORARY_TAB_KEY = 'temporary-tab';
 
+export const generateTempViewId = () => `_${Math.random().toString().substring(2, 7)}`;
```

Comment:
Moved in from `draggableTabBar.tsx` - this is a general function used by a lot of components. It should live in the context component not some random component in the hierachy